packages/lib/src/createSyntheticEvent/createSyntheticEvent.ts

```diff
@@ -12,10 +12,13 @@ export const createSyntheticEvent = <
 >(
   event: NativeEventType,
   target: TargetType,
+  key?: string,
```

Comment:
I would make the 3rd arg an object that we spread onto the return object. More extensible this way

Reply:
Ideally we'd convert the `target` arg into an object, but that's a breaking change